controller/network/path.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 type Path struct {
-	Nodes     []*Router
-	Links     []*Link
-	IngressId string
-	EgressId  string
+	Nodes               []*Router
+	Links               []*Link
+	IngressId           string
+	EgressId            string
+	TerminatorLocalAddr string
 }
 
```

Comment:
It looks like the formatting is off a bit. Can you check if you're running gofmt on save?

Reply:
I think maybe it's just the github view messing things up. If expand the section above the if block below, it shows the code indented 8, but that block shows indented 4. I'm guessing there's some difference in the GH formatter for expanded sections vs diff sections.

Also, not sure why all my comments got double posted :(